presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestPasswordAuthentication.java

```diff
@@ -19,6 +19,7 @@
 import com.google.common.collect.ImmutableMap;
```

Comment:
> "Move resource allocation out of test constructor"

Why?

Reply:
Because 

1. resources allocated in constructor are held during all tests' execution duration, not only when tests from given class are being executed 
2. failure to allocate them is nicely swallowed by surefire, so you do not know what happened

Reply:
Ok, please add that to the commit message

Reply:
added